gammapy/irf/edisp/tests/test_kernel.py

```diff
@@ -129,8 +126,5 @@ def test_io_ogip_checksum(tmp_path):
     path = tmp_path / "test.fits"
     edisp.write(path, checksum=True)
 
-    # TODO: why does this fail?
-    # MATRIX hdu checksum is changed.
-    with warnings.catch_warnings():
-        warnings.simplefilter("error")
-        fits.open(path, checksum=True)
+    edisp_2 = fits.open(path, checksum=True)
+    assert edisp_2[1].header["HDUCLASS"], "OGIP"
```

Comment:
Do we need this? This test is only for the checksum.
Also, the checksum problem has resolved?